client/signup/jetpack-connect/sso.jsx

```diff
@@ -483,4 +498,9 @@ export default connect(
 		authorizeSSO,
 		validateSSONonce
 	}, dispatch )
+);
+
+export default flowRight(
```

Comment:
When using just `localize` and `connect`, I prefer keeping not using `compose` or `flowRight` but keeping the call simpler:

```
export default connect(
	...
)( localize( JetpackSSOForm ) );
```

but I don't have a strong opinion on that, and I'm fine with the way you're using it here.